src/Tools/dotnet-monitor/ParameterCapturing/ParameterCapturingEvents.cs

```diff
@@ -88,8 +90,19 @@ public static class CapturedParameterPayloads
             public const int ParameterType = 3;
             public const int ParameterTypeModuleName = 4;
             public const int ParameterValue = 5;
-            public const int ParameterAttributes = 6;
-            public const int ParameterTypeIsByRef = 7;
+            public const int ParameterValueEvaluationFlags = 6;
+            public const int ParameterAttributes = 7;
+            public const int ParameterTypeIsByRef = 8;
+        }
+
+        [Flags]
+        public enum ParameterEvaluationFlags
```

Comment:
It looks like we don't use more than 1 flag at a given time (i.e. they are exclusive), let's make this a regular enum instead?